docs/releases/3.1.txt

```diff
@@ -67,16 +67,17 @@ It should have no noticeable effect on any existing Django projects.
 JSONField for all supported database backends
 ---------------------------------------------
 
-Django now includes a :class:`models.JSONField <django.db.models.JSONField>`
-and a :class:`forms.JSONField <django.forms.JSONField>` that can be used on all
+Django now includes the :class:`.models.JSONField` and
+:class:`forms.JSONField <django.forms.JSONField>` that can be used on all
 supported database backends. Both fields support the use of custom JSON
 encoders and decoders. The model field supports the introspection, lookups, and
 transforms that were previously PostgreSQL-only.
 
-If your project uses :class:`django.contrib.postgres.fields.JSONField`, plus
-the related form field and transforms, you should adjust to use the new field
-and create a migration. For now, the old fields and transforms are left as a
-reference to the new ones and are deprecated as of this release.
+If your project uses ``django.contrib.postgres.fields.JSONField``, plus the
+related form field and transforms, you should adjust to use the new fields,
+and generate and apply a database migration. For now, the old fields and
```

Comment:
By `generate`, do you mean running `makemigrations`? That would create unnecessary (and problematic) migrations, wouldn't it? Shouldn't the instructions be to "modify existing migrations so they point to `models.JSONField`"?